src/handlers.js

```diff
@@ -4,52 +4,65 @@ const DEFAULT_WRAPPERS = {
   tsnode: 'lambdaWrapper',
 };
 
-const WRAPPER_CODE = {
-  python: `
-from RELATIVE_PATH import METHOD as METHOD_internal
-METHOD = METHOD_internal
+const WRAPPER_CODE = ({relativePath, method, wrapper, token, appName, collectorUrl, metadataOnly, urlsToIgnore, ignoredKeys}) => {
+  const commonNode = `
+
+process.env.EPSAGON_URLS_TO_IGNORE = process.env.EPSAGON_URLS_TO_IGNORE || '${urlsToIgnore}'; 
```

Comment:
I would rather not introduce this environment variables in case the param is undefined, something like
`${urlsToIgnore ? 'process.env.EPSAGON_URLS_TO_IGNORE = ${urlsToIgnore}' : ''}`
(same in Python)

Reply:
We would have failed in those cases.  
Good catch! 🎣 
Fixed in https://github.com/epsagon/serverless-plugin-epsagon/pull/77/commits/26604f156ce124f5c16270d99cd3684ff1eb17f2